### ISSUE TITLE
canu 1.7.1 stops in correction

### ISSUE THREAD

**Issue by umdChenwaiting** [marbl/canu#1367]
Hi, I tried to run canu 1.7.1 on Cyverse Atmosphere, however not very successfully. Because I couldn't copy any text in the Atmosphere terminal, I show you the whole run in terms of pictures. Hopes that will work. Thanks in advance.

By the way, the estimated genomesize is 990m, while the size of input package was only 7.5GB, is that a enough depth to run?
![1](https://user-images.githubusercontent.com/42663082/57974547-1668f080-7988-11e9-8a41-ba957e9bceb5.png)
![2](https://user-images.githubusercontent.com/42663082/57974548-1668f080-7988-11e9-9fb2-4a5c4abfda04.png)
![3](https://user-images.githubusercontent.com/42663082/57974549-1668f080-7988-11e9-86c5-943a89b9998e.png)
![4](https://user-images.githubusercontent.com/42663082/57974550-1668f080-7988-11e9-8222-a2bcd20545b1.png)
![5](https://user-images.githubusercontent.com/42663082/57974551-1668f080-7988-11e9-803f-d48f3423326a.png)
![6](https://user-images.githubusercontent.com/42663082/57974552-1668f080-7988-11e9-81b7-28c468f130cb.png)
![7](https://user-images.githubusercontent.com/42663082/57974553-1668f080-7988-11e9-95bd-a0129c8d2504.png)
![8](https://user-images.githubusercontent.com/42663082/57974554-1668f080-7988-11e9-97e1-36242a81d8b6.png)
![9](https://user-images.githubusercontent.com/42663082/57974555-1668f080-7988-11e9-9f8d-e60968c4e33a.png)

**Comment by skoren** [comment]
I'd guess from the logs, you are running out of space, you only had 10gb free at the start and are down to about 5 at the end. 

However, given that you only have 60k reads for a 1gb genome, you're not going to get assembly or corrected reads. Even if the reads are 50kb on average (and I bet they're not) you have 3x coverage of the genome. More realistically, assuming the reads are 10kb you have about 0.5x. Neither one of those is going to assemble or correct anything but the few most repetitive parts of the genome.

**Comment by umdChenwaiting** [comment]
> I'd guess from the logs, you are running out of space, you only had 10gb free at the start and are down to about 5 at the end. I assume if you don't have the logs from the run you don't have the individual errors (like `correction/results/*4*.out`) either which would mean we can't find the exact cause of the error.
> 
> However, given that you only have 60k reads for a 1gb genome, you're not going to get assembly or corrected reads. From the logs, you have about 0.5x fold coverage. That isn't going to assemble or correct anything but the few most repetitive parts of the genome.

Thanks for your kind help. Given the size of my genome(1Gb), How many sequences do you recommond?

**Comment by skoren** [comment]
We recommend at least 20x coverage and higher if your genome is very heterozygous (see the FAQ). The longer your reads the better so the number of sequences doesn't matter as much as coverage.

**Comment by umdChenwaiting** [comment]
> We recommend at least 20x coverage and higher if your genome is very heterozygous (see the FAQ). The longer your reads the better so the number of sequences doesn't matter as much as coverage.

Hi agian. Thanks for your expaination. 
I came back to check the input file again. There are 280 fastq files, of each contains 4000 sequences, in the input compressed directory. However, as it indicated in the canu running, there were only just over 60000 input raw sequence. I do not know why I lost such a number of sequence when running. Do you have any idea about that?

**Comment by umdChenwaiting** [comment]
I calculated the number of the sequence this time before running. The number was over 1110000. However, canu also detected around 60000 this time.
![expected sequence number](https://user-images.githubusercontent.com/42663082/58126047-04818a80-7be0-11e9-9bae-03f7a7eae1a1.png)
![input sequence number](https://user-images.githubusercontent.com/42663082/58126048-04818a80-7be0-11e9-8a19-bcb9d5c7554e.png)
![running command](https://user-images.githubusercontent.com/42663082/58126049-04818a80-7be0-11e9-94e2-7959be34e8d3.png)

**Comment by skoren** [comment]
Canu will report all the files it loaded and any reads it filtered. post your r1.gkpStore.err and r1.gkpStore.ssi files.

**Comment by umdChenwaiting** [comment]
> Canu will report all the files it loaded and any reads it filtered. post your r1.gkpStore.err and r1.gkpStore.ssi files.

Thanks for your guide. I checked the r1.gkpStore.err, it says that the compressed file corrupt. I think that might be the reason that only partial of the sequnces were detected.

This time, I input the uncompressed file, so that the number of sequences might reach to 1million.  The running has stayed here for around 16 hours. since I started run at 20pm, 21, May. Is this a normal situation?
![微信截图_20190522123642](https://user-images.githubusercontent.com/42663082/58192539-f179c380-7c8e-11e9-9f66-9b50c963dd0f.png)
